linux/deploy_vm/deploy_vm_from_iso.yml

```diff
@@ -213,12 +203,80 @@
         vm_wait_log_delay: 30
         vm_wait_log_retries: 120
 
-    - name: "Wait 60s for OS rebooting"
-      ansible.builtin.pause:
-        seconds: 60
+    - name: "Take a screenshot when autoinstall complete message is received"
+      include_tasks: ../../common/vm_take_screenshot.yml
+      vars:
+        vm_take_screenshot_local_path: "{{ current_test_log_folder }}"
+
+    # VMware Photon OS kickstart file can't add shutdown action,
+    # so here needs to shutdown it separately
+    - name: "Shutdown VMware Photon OS VM after auto install completes"
+      when:
+        - unattend_install_conf is defined
+        - unattend_install_conf is match('Photon')
+      block:
+        - name: "Wait for guest full name on VMware Photon OS VM"
```

Comment:
Why get full name here? Does it need to be executed after power off/power on?

Reply:
Photon OS won't shutdown or reboot after autoinstall complete. Here adds a task to wait for guest full name is to ensure the guest OS is completely up as tools can collect guest full name successfully.